Source/Python/WarpXWrappers.cpp

```diff
@@ -33,7 +33,7 @@ namespace
         auto data =
             static_cast<amrex::Real**>(malloc((*num_boxes) * sizeof(amrex::Real*)));
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```